test/test_train_mp_imagenet.py

```diff
@@ -31,7 +31,7 @@
     '--ddp': {
         'action': 'store_true',
     },
-    '--ddp_pjrt': {
+    '--pjrt_distributed': {
```

Comment:
can we add comment here to explain how is `pjrt_distributed` different than `ddp`?